packages/firebase_core/firebase_core/example/macos/Podfile

```diff
@@ -0,0 +1,82 @@
+platform :osx, '10.12'
+
+# CocoaPods analytics sends network stats synchronously affecting flutter build latency.
+ENV['COCOAPODS_DISABLE_STATS'] = 'true'
+
+project 'Runner', {
+  'Debug' => :debug,
+  'Profile' => :release,
+  'Release' => :release,
+}
+
+def parse_KV_file(file, separator='=')
```

Comment:
These seem incredibly fluffy too, seeing other projects put these in a shared library and clear out the Podfile has been refreshing, might be a strategy here

Reply:
This is the generated podfile from Flutter :) not changed anything other than the version on the first line 🤓

Reply:
Oh that's right - I think I actually purged the Podfile from most of the locations since it was generated. Shame a generated file has to be checked in, but unless the podspec auto-edits macos min version constraint after generation I don't know what I could suggest. Makes sense